ml-proto/Makefile

```diff
@@ -10,7 +10,6 @@ NAME_UNOPT =    $(NAME)
 DIRS =		given host spec
 LIBS =		str bigarray
 
-OCB_FLAGS += 	-quiet
```

Comment:
Was moved to the individual targets below, since it must not be used for the run creating the batch file.